dmscripts/notify_suppliers_of_awarded_briefs.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime, timedelta
+
+from dmutils.email.exceptions import EmailError
+from dmutils.email.helpers import hash_string
+from dmutils.formats import DATE_FORMAT
+from dmscripts.helpers import env_helpers
+
+
+def create_context_for_brief(stage, brief):
```

Comment:
If the only reason you're not underscore-prefixing this function is because you're importing it for a test, _I'd argue_ you can still go ahead and prefix-underscore it. It'll make someone think twice before going and depending on it. And if someone _did_ decide to go and change it willy nilly as a result, the test would break, so that would be ok.